test/evm_script.js

```diff
@@ -326,37 +336,23 @@ contract('EVM Script', accounts => {
                 })
             })
 
-            context('registry', () => {
-                it('can be disabled', async () => {
+            context('> Registry actions', () => {
```

Comment:
Note that these tests have been simplified as most of the functionality was [already tested earlier](https://github.com/aragon/aragonOS/blob/2042452a3b248a56e98339e3e6cbccd6e888b2fc/test/evm_script.js#L92).